packages/web-components/src/divider/divider.styles.ts

```diff
@@ -0,0 +1,125 @@
+import { css } from '@microsoft/fast-element';
+import {
+  colorBrandForeground1,
+  colorBrandStroke1,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralStroke1,
+  colorNeutralStroke2,
+  colorNeutralStroke3,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontWeightRegular,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Divider styles
+ * @public
+ */
+export const styles = css`
+  :host {
+    display: flex;
+  }
+
+  :host::after,
+  :host::before {
+    align-self: center;
+    background: ${colorNeutralStroke2};
+    box-sizing: border-box;
+    content: '';
+    display: flex;
+    flex-grow: 1;
+    height: ${strokeWidthThin};
+  }
+
+  :host([inset]) {
+    padding: 0 12px;
+  }
+
+  :host ::slotted(*) {
+    color: ${colorNeutralForeground2};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    font-weight: ${fontWeightRegular};
+    margin: 0;
+    padding: 0 12px;
+  }
+
+  :host([align-content='start'])::before,
+  :host([align-content='end'])::after {
+    flex-basis: 12px;
+    flex-grow: 0;
+    flex-shrink: 0;
+  }
+
+  :host([orientation='vertical']) {
+    height: 100%;
+    min-height: 84px;
```

Comment:
Where does the `84px` come from?

Reply:
![image](https://user-images.githubusercontent.com/12279161/220450095-0ba64d8e-81ee-4a7e-860b-d69988df6201.png)